src/Helpers/TableToolbarHelper.js

```diff
@@ -128,5 +134,16 @@ export const removeFilters = (chips, apply) => {
 
         return obj;
     }, {});
+
     apply({ ...emptyFilter, page: 1 });
 };
+
+export const isFilterInDefaultState = (currentParams, defaultParams, filterParams) => {
+    // filter out params which have nothing to do with filter chips, like page, sort, etc.
+    currentParams = Object.keys(currentParams).filter(key => filterParams.includes(key)).reduce((obj, key) => {
```

Comment:
This one can be simplified a bit and we can avoid the loops `filters, includes...`. 
it might look unfamiliar and hard to read at first but I promise it get better over time :) 
```
currentParams = filterParams.reduce((acc, param) => (
        {
            ...acc, // return the previous 
            ...currentParams[param] && { [param]: currentParams[param] } // check if exists if it it return it 
        }
    ), {}); // I want to return object so I pass the initial `{}` and the acc will get it
```
The worst case the reduce will run `filterParams.length` times